packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/agreements/seniority/validator.ts

```diff
@@ -14,7 +17,7 @@ export const customSeniorityValidator = (
 ): any => {
   let idcc: SupportedCcIndemniteLicenciement | null = null;
   if (agreeement) {
-    idcc = `IDCC${agreeement.num}` as SupportedCcIndemniteLicenciement;
+    idcc = getSupportedAgreement(agreeement.num);
```

Comment:
J'ai homogénéisé pour utiliser la méthode `getSupportedAgreement` qui permet ainsi de valider que l'idcc est bien dans la liste